setup.py

```diff
@@ -55,7 +55,9 @@ def get_advanced_templates():
     entry_points={
         'console_scripts': [
             'release=release:main',
-            'ccm-deploy-test=test_util.test_installer_ccm:main',
+            # Remove this once CI is updated
+            'ccm-deploy-test=test_util.test_aws_vpc:main',
```

Comment:
Sure, I'll update the comment and remove the new endpoint. It just feels _wrong_ to have an executable called ccm-* that doesnt touch CCM :P